pkgs/by-name/dn/dnspy/deps.nix

```diff
@@ -0,0 +1,85 @@
+  (fetchNuGet { pname = "Microsoft.Diagnostics.DbgShim.win-arm"; version = "8.0.510501"; hash = "sha256-d1sTw6L4rrOW0WulyoEKAkH+bLj6wqd1vsHivVdlbO0="; })
+  (fetchNuGet { pname = "Microsoft.Diagnostics.DbgShim.win-arm64"; version = "8.0.510501"; hash = "sha256-jyhrx+x1jKUgNq0zgyte0f8gExiwjgqduj1fLppbaK4="; })
+  (fetchNuGet { pname = "Microsoft.Diagnostics.DbgShim.win-x64"; version = "8.0.510501"; hash = "sha256-2G/AFIR5/P5NFGzYyC4eQNl+9FZnrbxGjVf0VkQn4FU="; })
+  (fetchNuGet { pname = "Microsoft.Diagnostics.DbgShim.win-x86"; version = "8.0.510501"; hash = "sha256-t+L9LbwakGnx625hVKunpiyX+jV1abcemP5vHiHGRO4="; })
+  (fetchNuGet { pname = "Microsoft.Diagnostics.Runtime"; version = "1.1.142101"; hash = "sha256-6diZT/XBSfF2lpZrmcxvqEm8jRb5uAomoXTFiuZiJc8="; })
+  (fetchNuGet { pname = "Microsoft.DiaSymReader"; version = "2.0.0"; hash = "sha256-8hotZmh8Rb6Q6oD9Meb74SvAdbDo39Y/1m8h43HHjjw="; })
+  (fetchNuGet { pname = "Microsoft.NETCore.App.Host.win-x64"; version = "8.0.8"; hash = "sha256-pkfKvNeb779TUp9jp19peJjCXK3NGpexaFjWwc3dSBo="; })
+  (fetchNuGet { pname = "Microsoft.NETCore.App.Runtime.win-x64"; version = "8.0.8"; hash = "sha256-C0zUiMMAQ9nd1n7PDVoBhCShHzdGI67YFySTpeFH8uE="; })
+  (fetchNuGet { pname = "Microsoft.VisualStudio.Composition"; version = "17.10.37"; hash = "sha256-Lq0XlMb7eqfMsu+NsptjCZReU3vRH5+JGvPU1VbuyEY="; })
+  (fetchNuGet { pname = "Microsoft.VisualStudio.Composition.Analyzers"; version = "17.10.37"; hash = "sha256-pckf7uZKhMSLxGHtrR2P3qdeNTtEDjAJGvagQ0Oocww="; })
+  (fetchNuGet { pname = "Microsoft.VisualStudio.CoreUtility"; version = "15.5.27130"; hash = "sha256-mb1Xz+Nc5L2DF/9dQDzExO6VhJbT17jCQMa1QGaLxbM="; })
+  (fetchNuGet { pname = "Microsoft.VisualStudio.Language.Intellisense"; version = "15.5.27130"; hash = "sha256-B50voQegMnPUXV4MZKjsQ1svPo+uwYLugtD60qQnO00="; })
+  (fetchNuGet { pname = "Microsoft.VisualStudio.Text.Data"; version = "15.5.27130"; hash = "sha256-p19pt6/oiOVcQUJ2vd7uav+ef4E/2L3s/XURCQ6sPNg="; })
+  (fetchNuGet { pname = "Microsoft.VisualStudio.Text.Logic"; version = "15.5.27130"; hash = "sha256-O9KvdkpX1ikP5fZKfFGL+fcUFm1qPXv+ZtmmtwmDxaE="; })
+  (fetchNuGet { pname = "Microsoft.VisualStudio.Text.UI"; version = "15.5.27130"; hash = "sha256-RNcSVzMflgsTSHBO2qrtymQdsQJ9/1KQ+Ze+qIVXyio="; })
+  (fetchNuGet { pname = "Microsoft.VisualStudio.Text.UI.Wpf"; version = "15.5.27130"; hash = "sha256-WxeN3vcQu3mUiB/hTuB1Pm49bUx+FZQ3b4kGKfgdUQE="; })
+  (fetchNuGet { pname = "Microsoft.VisualStudio.Validation"; version = "17.8.8"; hash = "sha256-sB8GLRiJHX3Py7qeBUnUANiDWhyPtISon6HQs+8wKms="; })
+  (fetchNuGet { pname = "Microsoft.WindowsDesktop.App.Ref"; version = "8.0.8"; hash = "sha256-Gd3Y0mpvG9lADIr6o/E8Ab3Efml9RynKxVJCgBq+H9w="; })
+  (fetchNuGet { pname = "Microsoft.WindowsDesktop.App.Runtime.win-x64"; version = "8.0.8"; hash = "sha256-N78uhT3zWQIpl7TmVIuwhG7IIt0ti8+DUFGMxogOT7k="; })
```

Comment:
@corngood This is an implicit SDK reference when you enable `UseWpf`/`UseWindowsForms`, and it's not covered by `targetPackages` currently

Reply:
I think adding these to `targetPackages` (via the dotnet binary update script) would be reasonable.

Reply:
Done in 6cf9b69f3e09a1fc15957379fa0ddf75225b316b, but it will need to be run alongside a full dotnet update

Reply:
> it will need to be run alongside a full dotnet update

why do you say that? what you have looks good to me.

Reply:
oh, is it just because it still needs to be run on 6 and 9?

Reply:
> oh, is it just because it still needs to be run on 6 and 9?

Yeah, what I've done is `pkgs/development/compilers/dotnet/update.sh 8.0.8` and just copy pate the `targetPackages` (I couldn't copy the whole file because the sdks got bumped).
I wanted to do it for the others aswell but running update.sh with preview label didn't work so I thought it's going to be less effort to run it with/after #348077